iml-agent/src/action_plugins/postoffice.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::{agent_error::ImlAgentError, daemon_plugins::postoffice::CONF_FILE};
```

Comment:
This should still be an environment variable.

Reply:
Nope, I needs to be called from CLI, where the ENV doesn't exist

Reply:
Understood about ENV not being there. My point is that ENV *should* be there in the CLI.

Here's how we do so in the manager CLI which uses ENV:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/d0a29506e78353005c77900988d422d2c3c7ed9b/iml-manager-cli/src/main.rs#L52

We should lift all env vars into an environment var file and then do the same thing for the agent CLI.

Can you either do so in this patch or open a new issue for this and someone can pick it up.